src/Integration.Vsix/SonarLint.vsct

```diff
@@ -20,12 +20,47 @@
   <Include href="Commands\PackageCommands.vsct"/>
   <Include href="SonarLintDaemon\DaemonCommands.vsct"/>
 
+  <Commands package="guidSonarLintPackage">
+
+    <Groups>
+      <Group guid="guidSonarLintPackageCmdSet" id="SonarLintMenuGroup" priority="0x0600">
```

Comment:
It might be worth including `Ext` the names to give an indication of which VS menu the menus/groups relate to.